data/filedefs/filetypes.Swift.conf

```diff
@@ -0,0 +1,42 @@
+# For complete documentation of this file, please see Geany's main documentation
+[styling=C]
+
+[keywords]
+primary=associativity break case catch class continue convenience default deinit didSet do else enum extension fallthrough false final for func get guard if import in infix init inout internal lazy let mutating nil operator override postfix precedence prefix private public repeat required return self set static struct subscript super switch throws true try var weak where while willSet
+secondary=Array Bool Dictionary ErrorType Int Float Double Set String Tuple UnicodeScalar abs max min print
+# documentation keywords for javadoc
+doccomment=author deprecated exception param return see serial serialData serialField since throws todo version
+
```

Comment:
Could also add this for `"""multiline string"""` highlighting:
```ini
[lexer_properties]
lexer.cpp.triplequoted.strings=1
```

Reply:
Done.